refract-spec.md

```diff
@@ -30,14 +30,10 @@ The Refract Element contains three properties, `element`, `attributes`, and `con
         - (string)
         - (Element)
 
-- `attributes` (enum)
+- `attributes` (object)
 
     The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. It MAY be an array of elements, or it MAY be an object. The keys of the object SHOULD be defined element types with valid values for the corresponding element types.
 
-    - Members
-        - (object)
-        - array[Element]
-
 - `content` (enum)
 
     The `content` property defines the content of the instance of the specified element type. The value MAY be any of the JSON primitive types, an element, or an array of elements.
```

Comment:
I thought we were going to take out the reference to JSON here and just define these primitive types.